src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -42,6 +42,8 @@ public ClimbSubsystem() {
     m_rightPIDController.setSetpoint(-EXTEND_MOTOR_SETPOINT);
     m_rightClimbMotor.setIdleMode(IdleMode.kBrake);
     m_leftClimbMotor.setIdleMode(IdleMode.kBrake);
+    m_rightClimbMotor.setSmartCurrentLimit(EXTEND_CURRENT_LIMITS);
+   m_leftClimbMotor.setSmartCurrentLimit(RETRACT_CURRENT_LIMIITS);
```

Comment:
These should be using the same constant, since they are symmetrical. Your common constant could use the name CLIMB_CURRENT_LIMIT